lib/galaxy_test/workflow/multiple_text.gxwf-tests.yml

```diff
@@ -0,0 +1,14 @@
+- doc: |
+    Test that workflows can run with multiple text parameter fed into a multi select parameter.
+  job:
+    input:
+      value:
+      - --ex1
+      - ex2
+      - --ex3
```

Comment:
I'm confused by this syntax. If those are list items for `value`, shouldn't they be indented?
```yml
value:
  - --ex1
  - ex2
  - --ex3
```

Reply:
That's up to you, both serialise to a list, I think this is a matter of preference

Reply:
I see... ok, I just found it counterintuitive to me :sweat_smile:

Reply:
I've rebased this to add the additional indentation

Reply:
Thank you!